### PR TITLE
Add ScyllaDB

### LINKED ISSUE #9254: Add icon: ScyllaDB
### Brand Name

ScyllaDB

### Website

https://www.scylladb.com

### Popularity Metric

The ScyllaDB global rank is: 278,478
Total visits: 199.5K
See: https://www.similarweb.com/website/scylladb.com/#overview

### Official Resources for Icon and Color

svg: 
[ScyllaDB Logo](https://github.com/simple-icons/simple-icons/assets/68700461/01e0bf3b-6a8a-4064-99ab-00edcb9a380e)
Color: #6CD5E7 (The actual brand color and logo color)

### Additional Comments

Variants:
[Scylla-enterprise](https://github.com/simple-icons/simple-icons/assets/68700461/565305e4-4d96-4528-a04c-9f75945c725d)

[Scylla-manager](https://github.com/simple-icons/simple-icons/assets/68700461/2f12ea63-9b9e-4e06-9860-629a515dd921)

[Scylla-monitor](https://github.com/simple-icons/simple-icons/assets/68700461/5f674702-e597-4964-9dd8-3a32e3a810c9)

[Scylla-operator](https://github.com/simple-icons/simple-icons/assets/68700461/f713d61c-e17e-4805-af61-022c28baf7e0)

[Scylla-cloud](https://github.com/simple-icons/simple-icons/assets/68700461/8bb909a3-d0e1-4769-876a-331336ef2bf2)

[Scylla-drivers](https://github.com/simple-icons/simple-icons/assets/68700461/9b9f2164-30ba-40ad-9ef0-b4d665155f33)

### Contributing

Yes

### PR DESCRIPTION
**Pull request by Grifed-source**
**Issue:** closes #9254

**Similarweb rank:** 278,478. See https://www.similarweb.com/website/scylladb.com/#overview
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

SVG: Taken from https://www.scylladb.com/media-kit/
Color Taken from major color in logo `#6CD5E7`
The logo outline colour is: `#3D355C`

### CODE REVIEW

**Comment by Grifed-source** [comment]
Sorry for all the commits 💀😓

**Review by adamrusted — COMMENTED** [review]
We cannot use your branch as a source. 
The brand guidelines would also suggest we cannot use a custom treatment of the mascot.
Would you like to reach out to the ScyllaDB team directly, @Grifed-source?

**Comment by adamrusted** [comment]
Ah, fair enough. I would link to the media kit page as your source then - and it would appear there are [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/0178e355-2734-46c5-b3d8-5dcbd1f680c1) between your PR in black, and `logo-scylla-vertical-1-color-black-RGB.svg` in red.

**Comment by adamrusted** [comment]
> I'm talking to someone at ScyllaDB right now, i keep you updated

Amazing. See what they come back with.

**Comment by Grifed-source** [comment]
Hey I got the reply, they're allowing me to use the logo

**Comment by adamrusted** [comment]
Amazing. If you could refine based on the differences I found above, I'll look to get this one merged in.

File: _data/simple-icons.json
Comment (adamrusted) [inline _data/simple-icons.json:11120]: Source still needs updating to the media kit link. We can't use your fork of our repo as the source.

File: icons/scylladb.svg
Comment (adamrusted) [inline icons/scylladb.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ScyllaDB</title><path d="M18.782 2.53c-1.72-1.528-4.14-2.374-6.817-2.374C9.285.155 6.867 1 5.148 2.53c-1.728 1.539-2.68 3.716-2.68 6.118 0 2.315 2.326 13.236 2.42 13.703.165.717.795 1.21 1.504 1.21.086 0 .164-.009.25-.018a1.52 1.52 0 0 0 .424-.138c.294.13.622.182.942.121.242-.043.466-.147.656-.302a1.541 1.541 0 0 0 1.59.198c.225-.103.415-.259.562-.44a1.56 1.56 0 0 0 1.2.587 1.57 1.57 0 0 0 1.132-.501c.026-.026.052-.052.07-.078.242.294.596.502.976.553.38.06.778-.034 1.089-.25a.322.322 0 0 0 .086-.07c.18.148.397.251.622.294.164.035.328.044.493.026a1.44 1.44 0 0 0 .423-.112c.12.052.242.086.37.112a1.542 1.542 0 0 0 1.755-1.192c.095-.467 2.42-11.379 2.42-13.703.009-2.402-.942-4.57-2.67-6.117zM18.6 22.256c-.12.544-.64.907-1.192.847a1.53 1.53 0 0 0 .432-.812l.07-.415.267-1.65c.086-.553.173-1.106.242-1.66a.153.153 0 0 0-.112-.164.156.156 0 0 0-.182.113c-.12.544-.233 1.097-.345 1.641l-.32 1.642-.078.406a1.33 1.33 0 0 1-.12.32.922.922 0 0 1-.217.268 1.028 1.028 0 0 1-.285.19 1.432 1.432 0 0 1-.328.095 1.133 1.133 0 0 1-.752-.19c.104-.156.19-.329.233-.519.018-.043.018-.095.026-.138 0-.026.009-.052.009-.07l.008-.06.026-.241.052-.484.095-.968c.06-.648.12-1.296.164-1.944a.144.144 0 0 0-.13-.156.16.16 0 0 0-.172.13c-.095.64-.181 1.287-.268 1.927l-.12.967-.061.484c-.018.156-.035.337-.07.45-.06.259-.233.492-.457.639a1.08 1.08 0 0 1-.76.164 1.08 1.08 0 0 1-.796-.553c.026-.069.044-.138.06-.199.036-.138.036-.285.036-.397l.008-.717c.009-.475.017-.96.017-1.434.009-.476.009-.96 0-1.435 0-.077-.06-.147-.138-.147a.141.141 0 0 0-.155.139c-.026.475-.06.959-.078 1.434-.026.475-.043.959-.07 1.434l-.034.717c0 .06 0 .121-.008.173 0 .052-.01.095-.018.147-.026.095-.052.19-.103.277-.182.354-.57.587-.968.578a.865.865 0 0 1-.294-.043 1.462 1.462 0 0 1-.268-.112 1.123 1.123 0 0 1-.406-.423 1.041 1.041 0 0 1-.103-.277c-.026-.095-.026-.19-.035-.32l-.035-.717a79.64 79.64 0 0 0-.07-1.434l-.076-1.434a.15.15 0 0 0-.147-.139.146.146 0 0 0-.147.147v1.435c0 .475.008.959.017 1.434l.009.717c0 .112.008.26.034.397.018.07.035.139.06.208a1.14 1.14 0 0 1-.51.458 1.08 1.08 0 0 1-1.304-.337.96.96 0 0 1-.172-.32 1.381 1.381 0 0 1-.044-.181c-.008-.026-.008-.07-.017-.104l-.017-.112-.225-1.754a57.09 57.09 0 0 0-.25-1.754c-.01-.078-.078-.13-.156-.13a.148.148 0 0 0-.138.164c.043.588.095 1.175.147 1.763l.172 1.762.01.113c0 .034.008.069.016.12a1.554 1.554 0 0 0 .268.665c-.13.087-.276.157-.432.183a1.103 1.103 0 0 1-.665-.087 1.117 1.117 0 0 1-.493-.466 1.432 1.432 0 0 1-.12-.32l-.078-.406-.32-1.633c-.112-.545-.225-1.089-.346-1.633-.017-.078-.086-.13-.164-.113-.078.01-.138.087-.13.173.078.553.156 1.098.243 1.65l.267 1.642.07.415c.025.164.086.32.155.458.07.13.156.25.268.354-.017 0-.026.009-.043.009a1.097 1.097 0 0 1-1.244-.838c-.026-.113-2.411-11.336-2.411-13.609 0-2.28.898-4.328 2.523-5.788C7.04 1.408 9.355.604 11.92.604c2.567 0 4.882.803 6.515 2.264 1.633 1.451 2.523 3.507 2.523 5.788.052 2.273-2.333 13.496-2.36 13.6zm-4.406-11.172a1.292 1.292 0 0 0-.71-.397h-.033a.23.23 0 0 0-.242.207.23.23 0 0 0 .207.242.804.804 0 0 1 .346.104 10.887 10.887 0 0 1-3.612 1.581c-1.34.32-2.73.38-4.095.181a.187.187 0 0 0-.216.156.187.187 0 0 0 .156.216c.207.035.423.06.63.078 0 .008 0 .017.009.026.18.544.648 1.27 1.253 1.27.06 0 .12-.009.19-.026.224-.052.596-.285.596-1.218 0-.018-.01-.026-.01-.044a12.13 12.13 0 0 0 1.574-.242c.052-.008.104-.025.147-.034.12.484.302 1.089.907 1.089.25 0 .518-.139.726-.372.173-.199.449-.648.363-1.4a11.285 11.285 0 0 0 1.702-.95.82.82 0 0 1 .112.285.15.15 0 0 0 .164.121.148.148 0 0 0 .138-.164 1.157 1.157 0 0 0-.302-.709zm-5.962 2.428c0 .432-.103.743-.26.778-.336.086-.681-.415-.846-.795.372.017.735.017 1.106 0 .009.009 0 .017 0 .017zm3.465.104c-.112.13-.26.216-.38.216-.208 0-.33-.182-.475-.752a12.3 12.3 0 0 0 1.123-.389c.017.372-.078.7-.268.925zM9.744 5.07c-.717.147-.855 1.71-.432 2.143.285.285.856.371.856.57 0 .199-.57.285-.856.57-.432.432-.285 1.996.432 2.143.847.173 2.143-.856 2.143-2.713 0-1.858-1.296-2.886-2.143-2.713zm-.855 7.067a4.362 4.362 0 1 1 4.363-4.363 4.368 4.368 0 0 1-4.363 4.363Zm0-8.268a3.916 3.916 0 0 0-3.914 3.914 3.916 3.916 0 0 0 3.914 3.914 3.916 3.916 0 0 0 3.914-3.914A3.922 3.922 0 0 0 8.889 3.87Z"/></svg>
```

File: _data/simple-icons.json
Comment (adamrusted) [inline _data/simple-icons.json:11120]:
```suggestion
            "source": "https://www.scylladb.com/media-kit",
```

File: _data/simple-icons.json
Comment (adamrusted) [inline _data/simple-icons.json:11121]:
```suggestion
            "guidelines": "https://www.scylladb.com/media-kit"
```

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/scylladb.svg
Comment (adamrusted) [inline icons/scylladb.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ScyllaDB</title><path d="M18.782 2.53c-1.72-1.528-4.14-2.374-6.817-2.374C9.285.155 6.867 1 5.148 2.53c-1.728 1.539-2.68 3.716-2.68 6.118 0 2.315 2.326 13.236 2.42 13.703.165.717.795 1.21 1.504 1.21.086 0 .164-.009.25-.018a1.52 1.52 0 0 0 .424-.138c.294.13.622.182.942.121.242-.043.466-.147.656-.302a1.541 1.541 0 0 0 1.59.198c.225-.103.415-.259.562-.44a1.56 1.56 0 0 0 1.2.587 1.57 1.57 0 0 0 1.132-.501c.026-.026.052-.052.07-.078.242.294.596.502.976.553.38.06.778-.034 1.089-.25a.322.322 0 0 0 .086-.07c.18.148.397.251.622.294.164.035.328.044.493.026a1.44 1.44 0 0 0 .423-.112c.12.052.242.086.37.112a1.542 1.542 0 0 0 1.755-1.192c.095-.467 2.42-11.379 2.42-13.703.009-2.402-.942-4.57-2.67-6.117zM18.6 22.256c-.12.544-.64.907-1.192.847a1.53 1.53 0 0 0 .432-.812l.07-.415.267-1.65c.086-.553.173-1.106.242-1.66a.153.153 0 0 0-.112-.164.156.156 0 0 0-.182.113c-.12.544-.233 1.097-.345 1.641l-.32 1.642-.078.406a1.33 1.33 0 0 1-.12.32.922.922 0 0 1-.217.268 1.028 1.028 0 0 1-.285.19 1.432 1.432 0 0 1-.328.095 1.133 1.133 0 0 1-.752-.19c.104-.156.19-.329.233-.519.018-.043.018-.095.026-.138 0-.026.009-.052.009-.07l.008-.06.026-.241.052-.484.095-.968c.06-.648.12-1.296.164-1.944a.144.144 0 0 0-.13-.156.16.16 0 0 0-.172.13c-.095.64-.181 1.287-.268 1.927l-.12.967-.061.484c-.018.156-.035.337-.07.45-.06.259-.233.492-.457.639a1.08 1.08 0 0 1-.76.164 1.08 1.08 0 0 1-.796-.553c.026-.069.044-.138.06-.199.036-.138.036-.285.036-.397l.008-.717c.009-.475.017-.96.017-1.434.009-.476.009-.96 0-1.435 0-.077-.06-.147-.138-.147a.141.141 0 0 0-.155.139c-.026.475-.06.959-.078 1.434-.026.475-.043.959-.07 1.434l-.034.717c0 .06 0 .121-.008.173 0 .052-.01.095-.018.147-.026.095-.052.19-.103.277-.182.354-.57.587-.968.578a.865.865 0 0 1-.294-.043 1.462 1.462 0 0 1-.268-.112 1.123 1.123 0 0 1-.406-.423 1.041 1.041 0 0 1-.103-.277c-.026-.095-.026-.19-.035-.32l-.035-.717a79.64 79.64 0 0 0-.07-1.434l-.076-1.434a.15.15 0 0 0-.147-.139.146.146 0 0 0-.147.147v1.435c0 .475.008.959.017 1.434l.009.717c0 .112.008.26.034.397.018.07.035.139.06.208a1.14 1.14 0 0 1-.51.458 1.08 1.08 0 0 1-1.304-.337.96.96 0 0 1-.172-.32 1.381 1.381 0 0 1-.044-.181c-.008-.026-.008-.07-.017-.104l-.017-.112-.225-1.754a57.09 57.09 0 0 0-.25-1.754c-.01-.078-.078-.13-.156-.13a.148.148 0 0 0-.138.164c.043.588.095 1.175.147 1.763l.172 1.762.01.113c0 .034.008.069.016.12a1.554 1.554 0 0 0 .268.665c-.13.087-.276.157-.432.183a1.103 1.103 0 0 1-.665-.087 1.117 1.117 0 0 1-.493-.466 1.432 1.432 0 0 1-.12-.32l-.078-.406-.32-1.633c-.112-.545-.225-1.089-.346-1.633-.017-.078-.086-.13-.164-.113-.078.01-.138.087-.13.173.078.553.156 1.098.243 1.65l.267 1.642.07.415c.025.164.086.32.155.458.07.13.156.25.268.354-.017 0-.026.009-.043.009a1.097 1.097 0 0 1-1.244-.838c-.026-.113-2.411-11.336-2.411-13.609 0-2.28.898-4.328 2.523-5.788C7.04 1.408 9.355.604 11.92.604c2.567 0 4.882.803 6.515 2.264 1.633 1.451 2.523 3.507 2.523 5.788.052 2.273-2.333 13.496-2.36 13.6zm-4.406-11.172a1.292 1.292 0 0 0-.71-.397h-.033a.23.23 0 0 0-.242.207.23.23 0 0 0 .207.242.804.804 0 0 1 .346.104 10.887 10.887 0 0 1-3.612 1.581c-1.34.32-2.73.38-4.095.181a.187.187 0 0 0-.216.156.187.187 0 0 0 .156.216c.207.035.423.06.63.078 0 .008 0 .017.009.026.18.544.648 1.27 1.253 1.27.06 0 .12-.009.19-.026.224-.052.596-.285.596-1.218 0-.018-.01-.026-.01-.044a12.13 12.13 0 0 0 1.574-.242c.052-.008.104-.025.147-.034.12.484.302 1.089.907 1.089.25 0 .518-.139.726-.372.173-.199.449-.648.363-1.4a11.285 11.285 0 0 0 1.702-.95.82.82 0 0 1 .112.285.15.15 0 0 0 .164.121.148.148 0 0 0 .138-.164 1.157 1.157 0 0 0-.302-.709zm-5.962 2.428c0 .432-.103.743-.26.778-.336.086-.681-.415-.846-.795.372.017.735.017 1.106 0 .009.009 0 .017 0 .017zm3.465.104c-.112.13-.26.216-.38.216-.208 0-.33-.182-.475-.752a12.3 12.3 0 0 0 1.123-.389c.017.372-.078.7-.268.925zM9.744 5.07c-.717.147-.855 1.71-.432 2.143.285.285.856.371.856.57 0 .199-.57.285-.856.57-.432.432-.285 1.996.432 2.143.847.173 2.143-.856 2.143-2.713 0-1.858-1.296-2.886-2.143-2.713zm-.855 7.067a4.362 4.362 0 1 1 4.363-4.363 4.368 4.368 0 0 1-4.363 4.363Zm0-8.268a3.916 3.916 0 0 0-3.914 3.914 3.916 3.916 0 0 0 3.914 3.914 3.916 3.916 0 0 0 3.914-3.914A3.922 3.922 0 0 0 8.889 3.87Z"/></svg>
```

**Comment by Grifed-source** [comment]
I don't know why but when i commit your suggestions it adds a new line character

**Comment by adamrusted** [comment]
Weird. I'll see if I can fix it locally without making any material changes...

File: icons/scylladb.svg
Comment (adamrusted) [inline icons/scylladb.svg:1]: I'm still seeing a [difference](https://github.com/simple-icons/simple-icons/assets/1830867/cf8c97fc-b83e-4daa-bcf0-cade5d6d47d5) when comparing against `logo-scylla-vertical-1-color-black-RGB.svg` from the media kit. Could you please try re-generating against that? I can provide a path also, but that'd mean we have to wait for another maintainer to review. Up to you :)

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/scylladb.svg
Comment (adamrusted) [reply icons/scylladb.svg:1]: Below is my path, which matches source - and passes our linter. Feel free to use it @Grifed-source.
```html
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ScyllaDB</title><path d="M18.992 2.435C17.229.868 14.75 0 12.004 0 9.259 0 6.779.868 5.017 2.435 3.246 4.012 2.272 6.244 2.272 8.706c0 2.373 2.382 13.567 2.479 14.045.169.735.815 1.24 1.541 1.24.089 0 .168-.009.257-.018.151-.026.292-.07.434-.141.301.133.638.186.965.124.248-.045.479-.151.673-.31a1.579 1.579 0 0 0 1.63.203c.23-.106.425-.265.576-.451a1.596 1.596 0 0 0 1.231.602 1.596 1.596 0 0 0 .823-.239 1.695 1.695 0 0 0 .408-.354c.248.301.611.513 1 .566.39.062.797-.035 1.116-.256.036-.018.062-.045.089-.071.186.15.407.257.637.301.169.035.337.044.505.026.151-.017.292-.053.434-.115.124.053.248.089.381.115a1.58 1.58 0 0 0 1.798-1.222c.097-.478 2.479-11.663 2.479-14.045.009-2.462-.965-4.685-2.736-6.271Zm-.186 20.219c-.124.558-.655.93-1.222.868a1.59 1.59 0 0 0 .443-.833l.07-.425.275-1.691c.088-.567.177-1.134.248-1.701a.156.156 0 0 0-.115-.168.16.16 0 0 0-.186.115c-.124.558-.239 1.125-.354 1.683l-.328 1.683-.08.416c-.026.115-.071.23-.124.327a.947.947 0 0 1-.221.275 1.05 1.05 0 0 1-.292.195 1.512 1.512 0 0 1-.337.097 1.142 1.142 0 0 1-.77-.195c.106-.159.194-.336.239-.531.017-.044.017-.097.026-.142 0-.026.009-.053.009-.071l.009-.062.08-.743.097-.992c.062-.665.124-1.329.168-1.993a.147.147 0 0 0-.133-.159.164.164 0 0 0-.177.132c-.097.656-.186 1.32-.274 1.975l-.186 1.488c-.018.16-.036.346-.071.461a1.101 1.101 0 0 1-.469.655 1.104 1.104 0 0 1-.78.168 1.107 1.107 0 0 1-.814-.566c.026-.071.044-.142.061-.204.036-.142.036-.292.036-.408l.009-.735c.009-.487.017-.983.017-1.47a40.02 40.02 0 0 0 0-1.47c0-.079-.062-.15-.141-.15a.145.145 0 0 0-.16.141c-.026.487-.062.983-.079 1.47-.027.488-.045.983-.071 1.471l-.036.735c0 .062 0 .124-.008.177 0 .053-.009.097-.018.15-.027.098-.053.195-.106.284a1.11 1.11 0 0 1-.992.593.893.893 0 0 1-.301-.044c-.098-.027-.186-.071-.275-.115a1.154 1.154 0 0 1-.416-.434 1.082 1.082 0 0 1-.106-.284c-.027-.097-.027-.195-.036-.327l-.035-.735c-.018-.488-.045-.983-.071-1.471l-.08-1.47a.154.154 0 0 0-.15-.141.15.15 0 0 0-.151.15v1.47c0 .487.009.983.018 1.47l.009.735c0 .116.008.266.035.408.018.071.035.142.062.212a1.169 1.169 0 0 1-.523.47 1.109 1.109 0 0 1-1.337-.346 1.002 1.002 0 0 1-.177-.327 1.392 1.392 0 0 1-.044-.186c-.009-.027-.009-.071-.018-.107l-.018-.115-.23-1.797c-.08-.603-.159-1.205-.257-1.798-.009-.08-.079-.133-.159-.133-.089.009-.151.08-.142.168.044.602.098 1.205.151 1.807l.177 1.806.009.116c0 .035.009.07.017.124.009.088.036.177.062.256.045.151.124.293.213.426a1.129 1.129 0 0 1-1.125.097 1.154 1.154 0 0 1-.629-.806l-.079-.416-.328-1.674c-.115-.558-.23-1.116-.354-1.674-.018-.08-.089-.133-.168-.115-.08.009-.142.089-.133.177.079.567.159 1.125.248 1.692l.274 1.682.071.425c.027.169.089.328.16.47.07.133.159.257.274.363-.018 0-.027.009-.044.009a1.126 1.126 0 0 1-1.276-.859c-.026-.115-2.47-11.619-2.47-13.949 0-2.338.921-4.437 2.586-5.933C6.956 1.284 9.33.461 11.96.461s5.004.823 6.678 2.32c1.673 1.488 2.586 3.595 2.586 5.933.053 2.33-2.392 13.834-2.418 13.94Zm-4.517-11.451a1.322 1.322 0 0 0-.726-.407h-.035a.235.235 0 0 0-.248.212.235.235 0 0 0 .212.248.824.824 0 0 1 .354.106 11.147 11.147 0 0 1-3.701 1.621 11.15 11.15 0 0 1-4.198.186.193.193 0 1 0-.062.381c.212.035.434.062.646.08 0 .008 0 .017.009.026.186.558.664 1.302 1.284 1.302a.78.78 0 0 0 .195-.027c.23-.053.611-.292.611-1.248 0-.018-.009-.027-.009-.045a12.213 12.213 0 0 0 1.612-.248c.053-.008.106-.026.151-.035.124.496.31 1.116.93 1.116.256 0 .531-.142.744-.381.177-.204.46-.664.371-1.435a11.47 11.47 0 0 0 1.745-.974.834.834 0 0 1 .115.292c.018.08.089.133.169.124a.15.15 0 0 0 .141-.168 1.184 1.184 0 0 0-.31-.726Zm-6.11 2.489c0 .442-.107.761-.266.797-.346.088-.7-.426-.868-.815.381.018.753.018 1.134 0 .008.009 0 .018 0 .018Zm3.551.106c-.115.133-.266.221-.39.221-.212 0-.336-.186-.487-.77.39-.115.779-.248 1.151-.399.018.381-.079.718-.274.948Zm-2.878-1.515a4.471 4.471 0 1 1 4.472-4.472 4.476 4.476 0 0 1-4.472 4.472Zm0-8.475A4.014 4.014 0 0 0 4.84 7.82a4.014 4.014 0 0 0 4.012 4.012 4.013 4.013 0 0 0 4.011-4.012 4.02 4.02 0 0 0-4.011-4.012Zm.876 1.231c-.735.151-.876 1.754-.442 2.196.292.293.876.381.876.585 0 .204-.584.292-.876.584-.443.443-.293 2.046.442 2.197.868.177 2.197-.877 2.197-2.781 0-1.904-1.329-2.958-2.197-2.781Z"/></svg>
```

**Comment by adamrusted** [comment]
I think it might have something to do with editor config adding a new line. Let me update again to remove properly.

**Comment by Grifed-source** [comment]
image
it's one line but the lint is saying unexpected newline character

**Comment by adamrusted** [comment]
Can another member of @simple-icons/icon-review please review this one, now I've contributed?

**Review by service-paradis — APPROVED** [review]
LGTM, thanks to both of you 🎉

**Pull request merged**
